.test-infra/metrics/sync/github/sync_github_actions.py

```diff
@@ -0,0 +1,166 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
```

Comment:
Missing second half of the license here

Reply:
> Missing second half of the license here

Thanks for the heads up Kiley!
I just noticed that all the python scripts under that folder also lack the second half of the license.
I'll go ahead and add the missing half to them.